components/styled/icon/src/organisms/icon-button.tsx

```diff
@@ -25,6 +25,7 @@ const IconButton: FC<Props> = ({ iconProps, text, ...rest }) => (
 interface Props extends HTMLAttributes<HTMLButtonElement> {
   iconProps: IconProps
   text?: string
+  disabled?: boolean
```

Comment:
This is unnecessary as the disabled property is already provided on the HTMLButtonElement API - 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLButtonElement